Gemfile

```diff
@@ -1,8 +1,10 @@
 source 'https://rubygems.org'
 
 group :test do
-  gem 'cucumber', "=1.3.20"
-  gem 'aruba', "=0.6.1"
-  gem 'rspec', "=2.14.1"
+  gem 'gem-shut-the-fuck-up'
```

Comment:
It's for suppressing lengthy post-install messages from other gems (i've been tired of scrolling them all n-th time). I'll include it also in #93